dump-prog/dump.cpp

```diff
@@ -226,28 +249,35 @@ int printSecs(void *N,
               string &secName,
               image_section_header s,
               bounded_buffer *data) {
-  cout << "Sec Name: " << secName << endl;
-  cout << "Sec Base: 0x" << to_string<uint64_t>(secBase, hex) << endl;
+  static_cast<void>(N);
+  static_cast<void>(s);
+
+  std::cout << "Sec Name: " << secName << endl;
+  std::cout << "Sec Base: 0x" << to_string<uint64_t>(secBase, hex) << endl;
   if (data)
-    cout << "Sec Size: " << to_string<uint64_t>(data->bufLen, dec) << endl;
+    std::cout << "Sec Size: " << to_string<uint64_t>(data->bufLen, dec) << endl;
   else
-    cout << "Sec Size: 0" << endl;
+    std::cout << "Sec Size: 0" << endl;
   return 0;
 }
 
+#define DUMP_FIELD(x)                                                      \
```

Comment:
I have moved this here (it was previously declared inside the function) to unclutter the code a little